rll/upgrade.sh

```diff
@@ -1,64 +1,121 @@
-#! /bin/bash -e
-prefix=https://rightlinklite.rightscale.com/rll
-re="RLL ([^ ]*) "
-$RLBIN -version
-if [[ `$RLBIN -version` =~ $re ]]; then
-  current=${BASH_REMATCH[1]}
-else
+#!/bin/bash
+
+# Will compare current version of rightlink 'running' with latest version provided from 'upgrades'
+# file. If they differ, will update to latest version.  Note that latest version can be an older version
+# if a downgrade is best.
+
+# -e will immediatly exit out of script at point of error
+set -e
+
+RLL_SECRET=/var/run/rll-secret
+
+upgrade_rightlink() {
+
+  # Use logger here instead of echo since stdout from this is not sent to audit entries.
```

Comment:
Could we clarify this since stdout usually does actually go to audit entries and this is only the case because the script is restarting RightLink.